src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/UnnecessarySemicolonInEnumerationCheckTest.java

```diff
@@ -0,0 +1,70 @@
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2019 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.checks.coding;
+
+import static com.puppycrawl.tools.checkstyle.checks.coding.UnnecessarySemicolonInEnumerationCheck.MSG_SEMI;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.puppycrawl.tools.checkstyle.AbstractModuleTestSupport;
+import com.puppycrawl.tools.checkstyle.DefaultConfiguration;
+
+/**
+ * Test fixture for the UnnecessarySemicolonInEnumerationCheck.
+ *
+ */
+public class UnnecessarySemicolonInEnumerationCheckTest extends AbstractModuleTestSupport {
+
+    @Override
+    protected String getPackageLocation() {
+        return "com/puppycrawl/tools/checkstyle/checks/coding/"
+            + "unnecessarysemicoloninenumeration";
+    }
+
+    @Test
+    public void testDefault() throws Exception {
+        final DefaultConfiguration checkConfig =
+            createModuleConfig(UnnecessarySemicolonInEnumerationCheck.class);
+
+        final String[] expected = {
+            "24:12: " + getCheckMessage(MSG_SEMI),
+            "27:13: " + getCheckMessage(MSG_SEMI),
+            "30:14: " + getCheckMessage(MSG_SEMI),
+            "33:14: " + getCheckMessage(MSG_SEMI),
+            "36:54: " + getCheckMessage(MSG_SEMI),
+            "39:15: " + getCheckMessage(MSG_SEMI),
+            "42:56: " + getCheckMessage(MSG_SEMI),
+            "46:9: " + getCheckMessage(MSG_SEMI),
+            "51:33: " + getCheckMessage(MSG_SEMI),
+        };
+
+        verify(checkConfig, getPath("InputUnnecessarySemicolonInEnumeration.java"), expected);
+    }
+
+    @Test
+    public void testTokensNotNull() {
+        final UnnecessarySemicolonInEnumerationCheck check =
+                new UnnecessarySemicolonInEnumerationCheck();
+        Assert.assertNotNull("Acceptable tokens should not be null", check.getAcceptableTokens());
+        Assert.assertNotNull("Default tokens should not be null", check.getDefaultTokens());
+        Assert.assertNotNull("Required tokens should not be null", check.getRequiredTokens());
```

Comment:
What do we gain by asserting that these aren't `null`?
Wouldn't it be better to assert they are some actual value instead?

Reply:
This was taken from UnnecessaryParenthesesCheck to satisfy pitest.

Anyway, I changed it.